src/AclRegs.cc

```diff
@@ -162,8 +162,8 @@ Acl::Init()
     RegisterMaker("transaction_initiator", [](TypeName name)->ACL* {return new TransactionInitiator(name);});
 
 #if USE_LIBNETFILTERCONNTRACK
-    RegisterMaker("clientside_mark", [](TypeName name)->ACL* { return new Acl::ConnMark; });
-    RegisterMaker("client_connection_mark", [](TypeName name)->ACL* { return new Acl::ConnMark; });
+    RegisterMaker("clientside_mark", [](TypeName)->ACL* { return new Acl::ConnMark; }); // XXX: Add name parameter to ctor
+    RegisterMaker("client_connection_mark", [](TypeName)->ACL* { return new Acl::ConnMark; }); // XXX: Add name parameter to ctor
```

Comment:
I added those `XXX`s for consistency with other very similar lines in this file. If your think these marks should not be added, please remove them:

```suggestion
    RegisterMaker("clientside_mark", [](TypeName)->ACL* { return new Acl::ConnMark; });
    RegisterMaker("client_connection_mark", [](TypeName)->ACL* { return new Acl::ConnMark; });
```